library/psa_crypto_slot_management.c

```diff
@@ -371,8 +371,10 @@ psa_status_t psa_validate_key_location( psa_key_lifetime_t lifetime,
         return( PSA_SUCCESS );
 }
 
-psa_status_t psa_validate_key_persistence( psa_key_lifetime_t lifetime )
+psa_status_t psa_validate_key_persistence( psa_key_lifetime_t lifetime,
+                                           mbedtls_svc_key_id_t key_id )
 {
+    psa_status_t status;
```

Comment:
The CI is complaining about this declaration being unused if `MBEDTLS_PSA_CRYPTO_STORAGE_C` is undefined. Please move it inside the ifdef below.